src/components/CharacterLimitMainText/CharacterLimitMainText.spec.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import { render } from '@testing-library/react';
+
+import { networkListMock } from './networkListMock';
+
+import CharacterLimitMainText from './CharacterLimitMainText';
+
+test('renders CharacterLimitMainText', () => {
+  const { container } = render(
+    <CharacterLimitMainText
+      value={'hello mundo'}
```

Comment:
Nao tá errado, mas quando for texto nao precissa colocar entre brackets.

```suggestion
      value='hello mundo'
```

Reply:
workado